src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -446,11 +446,14 @@ class Settings extends Component {
 				lang: 'de-DE',
 				name: 'Deutsch'
 			}, {
-               			lang: 'am-AM',
-                		name: 'Armenian'
-            		}, {
+               	lang: 'am-AM',
```

Comment:
Not indented correctly, please fix. :smile:

Reply:
Actually, I fixed indentation as it was not correct in PR merged just before it. 😉